### ISSUE TITLE
Adapt Accessibility tests: cannot interact with Slider components using only keyboard

### ISSUE THREAD

**Issue by shauncg** [adaptlearning/adapt_framework#421]
It's not clear how the Slider component is supposed to be interacted with using keyboard navigation. The JAWS descriptive text doesn't offer any hints. 

Pressing the spacebar or enter key (the usual JAWS command/control keys) with the slider lozenge selected just opens the drawer (which is a bit odd - I'm not sure why this occurs).
